### PR TITLE
Convert overview tab to grid layout

### PR DESCRIPTION
**Pull request by jazeee**
Refactor first three columns to display grid.
Move a single usage of nowrap to arcane spells table.

## Screenies

| Old | New |
| ---- | ---- |
| image | image |

### CODE REVIEW

File: ui/components/gridTable/gridTable.scss [hunk 1]
```diff
@@ -33,7 +33,7 @@ $grid-column-gap: 2px;
   input[type=text] {
     text-align: left;
   }
-  input, span, select {
+  input, select {
```
Comment (jazeee) [inline ui/components/gridTable/gridTable.scss:36]: This was only used for IQ +-, so I put that specifically in the spells style.
The span was too liberal, and caused pain elsewhere.

File: ui/main.scss [hunk 2]
```diff
@@ -12,7 +12,6 @@
 @import './weightPenalties/weightPenalties.scss';
 @import './overviewTab/overviewTab.scss';
 @import './basicTab/basicTab.scss';
-@import './epicFeats/epicFeats.scss';
```
Comment (jazeee) [inline ui/main.scss:15]: These are now merged into `overviewTab.scss`

File: ui/overviewTab/overviewTab.pug [hunk 3]
```diff
@@ -1,43 +1,39 @@
...
+        .advantage-table(data-epic-table-name="advantage")
+          .conditional-something
+            input(name=advantage.fieldName type='hidden')
+            .advantage-row
+              div.advantage-name=advantage.description
+              span.advantage-value(name=advantage.fieldName)
       fieldset.repeating_extraadvantage
-        span.extraadvantagename(name='attr_extraadvantagename')
-    = '\n'
-    | &emsp;&emsp;
```
Comment (jazeee) [inline ui/overviewTab/overviewTab.pug:20]: These `emsp`'s are no longer needed.
`display: flex` + `gap: 16px` puts spacing between flex items.

File: ui/overviewTab/overviewTab.pug [hunk 4]
```diff
@@ -1,43 +1,39 @@
...
-      .table-header
+      .skills-table.epic-table-header-grid
         .skillname Skill
         .skillability Ability
         .skillbase Mod
       fieldset.repeating_skill
         // This input is used by CSS to hide disciplines
         input.isdiscipline(name='attr_skillisdiscipline' type='hidden')
-        .skill-overview-row-container
+        .skill-overview-row-container(data-epic-table-name="skills-table")
```
Comment (jazeee) [inline ui/overviewTab/overviewTab.pug:36]: I am probably going to go back to some of the first talbes and make them follow this pattern.
Explicit selectors are easier to manage.

File: ui/overviewTab/overviewTab.scss [hunk 5]
```diff
@@ -1,169 +1,162 @@
...
+      }
+    }
   }
 
-
-  .table-header, div[data-groupname] {
-    span {
-      display: inline-block;
+  /* ------------ Weapons Overview -------------- */
+  .weapons-overview {
```
Comment (jazeee) [inline ui/overviewTab/overviewTab.scss:68]: I haven't converted weapons yet.
It should be easy enough, but have to work through the notes block.

File: ui/spells/spells.html [hunk 6]
```diff
@@ -60,7 +60,7 @@
       <div class="skillTP-grid">TP</div>
       <div class="skillCP-grid">CP</div>
       <div class="skillexpertise-grid">Exper.</div>
-      <div class="skillattribute-grid">IQ ±</div>
+      <div class="skillattribute-grid arcane-iq-plus-minus-attribute">IQ ±</div>
```
Comment (jazeee) [inline ui/spells/spells.html:63]: This is specifically one column that I want to not word wrap. No visual change with this delta.
image

**Comment by jazeee** [comment]
Weapons grid will follow. (Just ran out of time.)

**Review by johnlamping — APPROVED** [review]

**Pull request merged**